app/controllers/admin/categories_controller.rb

```diff
@@ -10,6 +10,19 @@ def show
       }
     end
 
+    def new
+      render "admin/categories/new", locals: {
+        category: Category.new
+      }
+    end
+
+    def create
+      category_params = params.require(:category).permit(:name)
+      @category =  Category.new(category_params)
+      @category.save
```

Comment:
You can combine these two lines into `Category.create(category_params)`

Reply:
right away, dad